README.md

```diff
@@ -1,81 +1,27 @@
-# 1. Overview
+A crypto trading app demo to show a real-time trading dashboard for three different exchanges.
 
-Demo to show a real-time trading dashboard for three different exchanges.
+![crypto-app-ui](https://user-images.githubusercontent.com/1088136/198411981-2afbdd21-d145-4ad6-aea1-ee933ed482a5.png)
 
 The complete crypto-trading demo has two components:
 
-1. A stream application.
-2. A UI application written in Reactjs
+1. A backend built with Macrometa using a Document Collection, multiple Streams, and a Stream Worker.
 
+If you haven't already configure the backend, refer to the [getting started guide](https://macrometa.com/docs/apps/crypto-trading).
 
-The UI then makes use of all the streams and the `trades` collection to show the charts and suggestions at one place.
+2. A headless frontend written in ReactJS.
 
-For each of the three exchanges `USD`, `EUR` and `JPY`, this demo makes use of the following streams:
+The below steps will describe on how to set up the frontend.
 
-- `CrytoTraderQuotes{USD/EUR/JPY}`
-- `CryptoTraderQuotesAvg{USD/EUR/JPY}`
+Prerequisite: `nodejs` and `npm` must be installed on your system.
 
-The below steps will describe on how to deploy the node and UI application.
+## 1. Clone or Fork the Repo
 
-# 2. Prerequisites
+`git clone git@github.com:Macrometacorp/tutorial-cryptotrading.git` or click on the `Fork` button in the upper right hand corner of this repo.
```

Comment:
Tell the user what to do with this, i.e. "Run this command in your terminal" or click...